### ISSUE TITLE
deprecated filtering settings

### ISSUE THREAD

**Issue by xblitz** [sensu/sensu#1530]
the settings of the "deprecated filtering" should have a global server setting

## Expected Behavior
Be able to set a global server setting to disable the deprecated filtering

## Current Behavior
currently the settings needs to be in many places to completely bypass de deprecated filtering

- in checks
- in client config for the keepalive event
- in all manual tcp socket events

## Your Environment
* Sensu version used: 0.26.3

**Comment by portertech** [comment]
This is not a Sensu Core issue, but a sensu-plugin library one https://github.com/sensu-plugins/sensu-plugin

The deprecated filtering messages are only a temporary measure to notify users of impending changes. I for one am not against the idea of adding a "global" sensu-plugin configuration setting.

@xblitz I have created a sensu-plugin issue https://github.com/sensu-plugins/sensu-plugin/issues/166

Closing this one.
